app/Actions/Photo/Pipes/Standalone/CreateSizeVariants.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Actions\Photo\Pipes\Standalone;
+
+use App\Contracts\Models\SizeVariantFactory;
+use App\Contracts\PhotoCreate\StandalonePipe;
+use App\DTO\PhotoCreate\StandaloneDTO;
+use App\Exceptions\Handler;
+
+class CreateSizeVariants implements StandalonePipe
+{
+	public function handle(StandaloneDTO $state, \Closure $next): StandaloneDTO
+	{
+		// Create remaining size variants if we were able to successfully
+		// extract a reference image above
```

Comment:
```suggestion
		// extract a reference image
```
Only because it's not really "above" from here